README.md

```diff
@@ -56,7 +63,15 @@ curl https://cdn.rawgit.com/pimterry/notes/v0.2.0/notes._notes > /usr/share/zsh/
 
 You'll need to open a new shell for this to take effect.
 
-# How do I configure this?
+## What if I want to uninstall this?
+If you used the automated install script to install notes, you can uninstall it the same way by running:
+```bash
+curl https://rawgit.com/pimterry/notes/master/install.sh | bash -s uninstall
```

Comment:
Both this and the other curl above need to be `curl -L`. I think this is a new thing, but `rawgit.com` now includes a redirect, which curl doesn't follow by default.